test/externalTests.sh

```diff
@@ -41,6 +41,11 @@ DIR=$(mktemp -d)
     echo "Running Zeppelin tests..."
     git clone --depth 1 https://github.com/OpenZeppelin/zeppelin-solidity.git "$DIR"
     cd "$DIR"
+    # Fix some things that look like bugs
+    sed -i -e 's/let token = await ERC827TokenMock.new();/let token = await ERC827TokenMock.new(accounts[0], 0)/;' test/token/ERC827/ERC827Token.js
```

Comment:
I think this line should actually be removed in the source?

Reply:
why?

Reply:
if you remove the line, you essentially disable the test. if you modify it seems to fix the test...

Reply:
`beforeEach` creates the `token`. This given test sets the `allowance` for `accounts[1]` on said token.

Reply:
ok, sounds good!

Reply:
I did test it locally before suggesting it.